### PR TITLE
feat: Adding PointCloud Feature

### PR DESCRIPTION
**Pull request by Kerwin-L**
- feat: PointCloud2d supports swtching & projection
- feat: PointCloud supports for adding attribute & subAttribute

### CODE REVIEW

File: packages/lb-components/src/components/pointCloudView/PointCloud2DView.tsx [hunk 1]
```diff
@@ -1,15 +1,138 @@
...
+    if (topViewInstance && mappingData) {
+      const { pointCloudInstance } = topViewInstance;
+      const newAnnotations2d: IAnnotationDataTemporarily[] = [];
+      const defaultViewStyle = {
+        fill: 'transparent',
+        color: 'green',
+      };
+      pointCloudBoxList.forEach((pointCloudBox) => {
+        const viewDataPointList = pointCloudInstance.lidar2image(pointCloudBox, mappingData.calib);
+        viewDataPointList.forEach((v: any) => {
```
Comment (Glenfiddish) [inline packages/lb-components/src/components/pointCloudView/PointCloud2DView.tsx:61]: prefer to use map

File: packages/lb-components/src/components/pointCloudView/PointCloud2DView.tsx [hunk 2]
```diff
@@ -1,15 +1,138 @@
...
       </div>
     </PointCloudContainer>
   );
 };
 
-export default PointCloud2DView;
+const mapStateToProps = (state: AppState) => {
+  const { imgList, imgIndex } = state.annotation;
+
+  console.log('imgList[imgIndex]', imgList[imgIndex], imgList);
```
Comment (Glenfiddish) [inline packages/lb-components/src/components/pointCloudView/PointCloud2DView.tsx:132]: no console

File: packages/lb-components/src/types/data.d.ts [hunk 3]
```diff
@@ -1,10 +1,23 @@
 import { ESubmitType } from '@/constant';
+import { TMatrix13Tuple, TMatrix14Tuple } from '@labelbee/lb-utils';
 
 /** 标注文件对象 */
+export interface ICalib {
```
Comment (Glenfiddish) [inline packages/lb-components/src/types/data.d.ts:5]: comment for ICalib to explain P R T

File: packages/lb-components/src/components/pointCloudView/PointCloud2DView.tsx (hunk 1 above)
Comment (Kerwin-L) [reply packages/lb-components/src/components/pointCloudView/PointCloud2DView.tsx:61]: why map?

File: packages/lb-components/src/types/data.d.ts (hunk 3 above)
Comment (Kerwin-L) [reply packages/lb-components/src/types/data.d.ts:5]: I will add a document to describe the pointCloud Terminology

File: packages/lb-components/src/components/pointCloudView/PointCloud2DView.tsx (hunk 2 above)
Comment (Kerwin-L) [reply packages/lb-components/src/components/pointCloudView/PointCloud2DView.tsx:132]: It will be deleted later

**Pull request merged**